pkg/config/config.go

```diff
@@ -543,10 +544,52 @@ func SHA256(text string) string {
 	return hex.EncodeToString(algorithm.Sum(nil))
 }
 
+// parseCherryPicksToExternalRefs parses cherry-pick entries and converts them
+// to SPDX ExternalRef entries with category "OTHER" and type "cherry-pick".
+// The format expected is: [branch/]commit-id: comment
+// Each cherry-pick is represented as an ExternalRef with the locator being the
+// branch/commit format
+func parseCherryPicksToExternalRefs(cherryPicks string) []spdx.ExternalRef {
+	lines := strings.Split(cherryPicks, "\n")
+	refs := make([]spdx.ExternalRef, 0, len(lines))
+	for _, line := range lines {
+		line = strings.Split(line, "#")[0]
+		line = strings.TrimSpace(line)
+		if line == "" {
```

Comment:
```suggestion
		if line == "" || strings.HasPrefix(line, "#") {
```

IIUC we can also have comments in the cherry-pick list, which should be ignored as well.